type-definitions/most.d.ts

```diff
@@ -0,0 +1,144 @@
+declare type SeedValue<S, V> = { seed: S, value: V };
+declare type TimeValue<V>    = { time: number, value: V };
+
+declare interface Promise<A> {}
```

Comment:
Is it possible to reference [the type definition for es6-promise](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/es6-promise/es6-promise.d.ts) without actually listing es6-promise as a dep and without committing its type def file to the most repo? It'd be nice so that callers get the benefit of most.js having well-typed promise values.